index.bs

```diff
@@ -1838,6 +1839,18 @@ a [=header value=] or null |webTriggerHeader|, a [=header value=] or null
 
      </dl>
 
+To <dfn export>get the first non-coalesced header</dfn> given a [=response/header list=] |headers|,
+and a [=header name=] header:
+1. Let |non-coalesced-headers| be the result of [=get non-coalesced headers=] with |headers| and |header|.
+1. If |non-coalesced-headers| [=list/is empty=], return null.
+1. Return |non-coalesced-headers|[0].
+
+To <dfn export>get non-coalesced headers</dfn> given a [=response/header list=] |headers|,
```

Comment:
I don't think we want to export these algorithms; ideally they'd ultimately be part of the `fetch` spec itself.